CIScripts/Deploy/Deployment.ps1

```diff
@@ -17,15 +17,18 @@ function Install-Artifacts {
     Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
            [Parameter(Mandatory = $true)] [string] $ArtifactsDir)
 
+    function Test-NonemptyDir {
```

Comment:
IMHO this should be moved to Common/ dir

Reply:
I don't feel like creating a separate file for a one-liner function. Is there any file I can add it to?

Reply:
I guess we'll see if some other part of the codebase requires this function.